From f12ccf059ac93ca21b435c0e8c1d5bbb350b3a9e Mon Sep 17 00:00:00 2001
From: Andrea Suisani <sickpig@gmail.com>
Date: Mon, 5 Feb 2018 17:03:29 +0100
Subject: [PATCH] Use Logging namespace to access LogAcceptCategory

LogAcceptCategory and the enum used to define available logging
category is defined inside the Logging namespace. If used directly
outside of util.{h,cpp} reference to such namespace is needed.

This commit fix issue #950
---
 src/httpserver.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/httpserver.cpp b/src/httpserver.cpp
index 911a4a87d365..54faafd2ccaf 100644
--- a/src/httpserver.cpp
+++ b/src/httpserver.cpp
@@ -410,7 +410,7 @@ bool InitHTTPServer()
 #if LIBEVENT_VERSION_NUMBER >= 0x02010100
     // If -debug=libevent, set full libevent debugging.
     // Otherwise, disable all libevent debugging.
-    if (LogAcceptCategory(LIBEVENT))
+    if (LogAcceptCategory(Logging::LIBEVENT))
         event_enable_debug_logging(EVENT_DBG_ALL);
     else
         event_enable_debug_logging(EVENT_DBG_NONE);
